### DISCUSSION TITLE
Fialment returns 500 server errors

### DISCUSSION THREAD

**Discussion by StaffCollab** [filamentphp/filament#6557] [Help]
Hi All.

I have a Laravel/Filament project. Working great on the local server and working great on my forge server. But when I upload this to a cPanel server (shared hosting by liquid web -- my client's choice...), the project still loads, and even livewire and tailwind are working as expected, but any page containing a filament form or table is not loading.

I cannot understand why in the world this should happen.

The only special configuration that was needed to get Laravel working in cPanel was to move the public folder out of the laravel project and into the public_html folder and update the index.php with the relative path to the Laravel project.
These changes are working, as I can see from the fact that everything except the filament works.

That leaves me without a clue why this is not working.

Any help would be greatly appreciated.

**Comment by ManojKiranA** [comment]
Pls check your error log or set APP_DEBUG to true to view the actual error in the application.

**Reply by StaffCollab** [reply]
Already did that.

The weird thing is that even with debug set to true, env is set to local. Still, no error shows.
only the page shows
~~~
This page isn’t working
my-domain.com is currently unable to handle this request.
HTTP ERROR 500
~~~

**Reply by danharrin** [reply]
We cannot help unless we know the error.

**Comment by radumro** [comment]
In my case i enabled php-fpm and it started working
